### PR TITLE
Add `README` to `bullet_train-api` gem

### PR DESCRIPTION
**Pull request by newstler**
Here's my attempt to write the `README` for `bullet_train-api` gem.

### CODE REVIEW

File: bullet_train-api/README.md [hunk 1]
```diff
@@ -1,28 +1,131 @@
...
+
+#### Push OpenAPI document to Redocly
+
+Needs `REDOCLY_ORGANIZATION_ID` and `REDOCLY_API_KEY` to be set:
+
+    rake bullet_train:api:push_to_redocly
+
+#### Create separate translations for API
+
+Starting 1.6.27, Bullet Train Scaffolding generates separate translations for API documentation: `api_title` and `api_description`.
```
Comment (gazayas) [inline bullet_train-api/README.md:114]: "Starting in 1.6.27"

File: bullet_train-api/README.md [hunk 2]
```diff
@@ -1,28 +1,131 @@
...
+defined in the corresponding controller to generate the Parameters section in the schema.
+
+#### Examples
+
+In order to generate example requests and responses for the documentation in the
+`automatic_components_for` calls, the bullet_train-api gem contains `ExampleBot`
+which uses FactoryBot to build an in-memory representation of the model,
+then generates the relevant OpenAPI schema for that model.
+
+ExampleBot will attempt to create a instance of the given model called `<model>_example`.
```
Comment (gazayas) [inline bullet_train-api/README.md:61]: "an instance"

**Review by gazayas — COMMENTED** [review]
Thanks for writing this up! I just wrote a couple of comments to fix in the README.

**Comment by newstler** [comment]
@gazayas Thanks! That's still work in progress, I plan to slowly gather all the information possible in next 1-2 weeks. Feel free to add anything you think worth mentioning in README!

**Comment by jagthedrummer** [comment]
@newstler, just wondering if we should go ahead and merge this. In my opinion publishing documentation as it's in progress is better than keeping it hidden until it's 100% perfect. There seems to be a _lot_ of good info here that's otherwise completely unavailable to people. I'd love to get it out as soon as we can, even if we need to tweak it again later.

**Comment by newstler** [comment]
Hey @jagthedrummer,

I totally agree with you. Unfortunately didn't have time to finish that. Let me run my eyes again quickly on what could be fixed fast and let's merge it.

**Comment by newstler** [comment]
@jagthedrummer Just looked at it, feels pretty solid without noticeable dangling phrases, and I don't really remember what I wanted to add more. Rebased it over `main` and added a small detail. From me it's ready to be merged.

**Pull request merged**